pkg/gadgets/tcpconnect/tracer/standard/tracer.go

```diff
@@ -53,10 +55,14 @@ func NewTracer(config *tracer.Config, resolver containercollection.ContainerReso
 		eventCallback(event)
 	}
 
-	baseTracer, err := gadgets.NewStandardTracer(lineCallback,
+	mountNsMapPath := filepath.Join(gadgets.PinPath, uuid.New().String())
+	if err := config.MountnsMap.Pin(mountNsMapPath); err != nil {
```

Comment:
Is the pinning not already done in NewStandardTracer()?

Reply:
ditto. :D